### DISCUSSION TITLE
TuneIn Station State for different channel

### DISCUSSION THREAD

**Discussion by IT-Lerch** [hklages/node-red-contrib-sonos-plus#210] [Q&A] [answered]
Where can I find the ID of my radio station? In your Wiki its s24896 for SWR3 but i would like to listen to Energy.
THX

**Comment by hklages** [comment, marked as answer]
Hi.

You can add the station to My Sonos in the SONOS app and then use
group.play.mysonos with payload title

or

Play the station and use group.get.trackplus to get the radio id

Let me know if that works for you

**Reply by IT-Lerch** [reply]
Thanks buddy. Great Work!
